utils/constants.py

```diff
@@ -18,3 +18,22 @@
     "GRANDMASTER": "GM",
     "CHALLENGER": "CH",
 }
+TIER_VALUE = {
+    "UNRANKED": 3,
+    "IRON": 1,
+    "BRONZE": 2,
+    "SILVER": 3,
+    "GOLD": 4,
+    "PLATINUM": 5,
+    "DIAMOND": 6,
+    # master, grandmaster, challenger only have lp points starting 0lp for master,
+    # around 300lp for grandmaster and 650lp for master
+    # these are not fixed lp points as they are assigned to GM or Challenger
+    # depending on the ranking not based on how much lp points they have
+    # Challenger: 1 ~ 300, GrandMaster: 301 ~ 1000
+    "MASTER": 7,
+    "GRANDMASTER": 7,
+    "CHALLENGER": 7,
```

Comment:
API doesn't provide 'Master' / GrandMaster or Challenger and only provides lp values for these tiers??